libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/AndroidSpeechPlayer.java

```diff
@@ -131,19 +128,8 @@ private void initializeWithLanguage(Locale language) {
     textToSpeech.setLanguage(language);
   }
 
-  private void fireInstructionListenerIfApi14() {
-    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1) {
-      voiceListener.onStart();
-    }
-  }
-
   private void setVoiceListener(final VoiceListener voiceListener) {
     this.voiceListener = voiceListener;
-
-    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1) {
-      textToSpeech.setOnUtteranceCompletedListener(new Api14UtteranceListener(voiceListener));
```

Comment:
We should remove `Api14UtteranceListener` as it's not used anymore.

Reply:
done